2025-08-03 21:48:17,680 [INFO] logging in using static token
2025-08-03 21:48:18,479 [INFO] Shard ID None has connected to Gateway (Session ID: 80652aa5457b95096e18e08cc714d545).
2025-08-03 21:48:20,487 [ERROR] Ignoring exception in on_ready
Traceback (most recent call last):
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/client.py", line 481, in _run_event
    await coro(*args, **kwargs)
  File "/mnt/c/Users/fleet/OneDrive/Documents/coding/discord_bot/guildGateKeeper/bot.py", line 386, in on_ready
    await prompt_for_class_role(member)
  File "/mnt/c/Users/fleet/OneDrive/Documents/coding/discord_bot/guildGateKeeper/bot.py", line 325, in prompt_for_class_role
    if user_record.get("class_assigned"):
AttributeError: 'bool' object has no attribute 'get'
2025-08-03 22:05:51,778 [INFO] logging in using static token
2025-08-03 22:05:52,622 [INFO] Shard ID None has connected to Gateway (Session ID: 4a0d95e80a79746a425ba99ef725ce15).
2025-08-03 22:05:54,706 [ERROR] Ignoring exception in on_ready
Traceback (most recent call last):
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/client.py", line 481, in _run_event
    await coro(*args, **kwargs)
  File "/mnt/c/Users/fleet/OneDrive/Documents/coding/discord_bot/guildGateKeeper/bot.py", line 433, in on_ready
    await prompt_for_class_role(member)
  File "/mnt/c/Users/fleet/OneDrive/Documents/coding/discord_bot/guildGateKeeper/bot.py", line 372, in prompt_for_class_role
    if user_record.get("class_assigned"):
AttributeError: 'bool' object has no attribute 'get'
2025-08-03 22:08:25,928 [INFO] logging in using static token
2025-08-03 22:08:26,700 [INFO] Shard ID None has connected to Gateway (Session ID: dc76817d82c35c039f7a644110ee8998).
2025-08-03 22:15:20,537 [INFO] logging in using static token
2025-08-03 22:15:21,371 [INFO] Shard ID None has connected to Gateway (Session ID: a43f6d91dc591eba0d9d7100112e5915).
2025-08-03 23:47:55,837 [INFO] logging in using static token
2025-08-03 23:47:56,616 [INFO] Shard ID None has connected to Gateway (Session ID: a37561e311dad9aeb8bcd091f63856b8).
2025-08-03 23:53:19,580 [INFO] logging in using static token
2025-08-03 23:53:20,386 [INFO] Shard ID None has connected to Gateway (Session ID: 8cea10bcf6060dad6cc407ed4c4c8fad).
2025-08-04 01:12:24,187 [INFO] Shard ID None has successfully RESUMED session 8cea10bcf6060dad6cc407ed4c4c8fad.
2025-08-04 01:55:13,393 [INFO] Shard ID None has successfully RESUMED session 8cea10bcf6060dad6cc407ed4c4c8fad.
2025-08-04 02:28:44,721 [INFO] Shard ID None has successfully RESUMED session 8cea10bcf6060dad6cc407ed4c4c8fad.
2025-08-04 03:18:56,124 [INFO] Shard ID None has successfully RESUMED session 8cea10bcf6060dad6cc407ed4c4c8fad.
2025-08-04 03:27:52,197 [INFO] Shard ID None has successfully RESUMED session 8cea10bcf6060dad6cc407ed4c4c8fad.
2025-08-04 05:52:27,341 [INFO] Shard ID None has successfully RESUMED session 8cea10bcf6060dad6cc407ed4c4c8fad.
2025-08-04 06:31:55,434 [INFO] Shard ID None has successfully RESUMED session 8cea10bcf6060dad6cc407ed4c4c8fad.
2025-08-04 08:02:57,178 [INFO] Shard ID None has successfully RESUMED session 8cea10bcf6060dad6cc407ed4c4c8fad.
2025-08-04 10:12:08,898 [INFO] logging in using static token
2025-08-04 10:12:09,752 [INFO] Shard ID None has connected to Gateway (Session ID: 1632b605bb25c8ab1f5fcde0702ce88d).
2025-08-04 10:12:20,433 [INFO] logging in using static token
2025-08-04 10:12:22,377 [INFO] Shard ID None has connected to Gateway (Session ID: 08fd927f15dd19db3ac1aae1a110ee66).
2025-08-04 10:22:02,592 [INFO] logging in using static token
2025-08-04 10:22:03,548 [INFO] Shard ID None has connected to Gateway (Session ID: dc7a257b5cd29033b132feffd9edaf62).
2025-08-04 10:23:13,650 [ERROR] Ignoring exception in command setmainfor
Traceback (most recent call last):
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 1021, in invoke
    await self.prepare(ctx)
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 938, in prepare
    await self._parse_arguments(ctx)
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 845, in _parse_arguments
    transformed = await self.transform(ctx, param, attachments)
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 715, in transform
    return await run_converters(ctx, converter, argument, param)  # type: ignore
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/converter.py", line 1444, in run_converters
    return await _actual_conversion(ctx, converter, argument, param)
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/converter.py", line 1326, in _actual_conversion
    return await converter().convert(ctx, argument)
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/converter.py", line 281, in convert
    raise MemberNotFound(argument)
discord.ext.commands.errors.MemberNotFound: Member "@Terani" not found.
2025-08-04 10:35:11,111 [INFO] logging in using static token
2025-08-04 10:35:12,094 [INFO] Shard ID None has connected to Gateway (Session ID: 65049ddfb0527585006e8404359787b6).
2025-08-04 10:36:14,184 [INFO] Main for user 'Terani' (ID: 606266945460764727) set to 'Demani' by Bingtoolbar
2025-08-04 10:37:07,280 [INFO] Removed alt 'Terani' from user ID 606266945460764727
2025-08-04 10:37:07,282 [INFO] Alt 'Terani' (Mage) reassigned to user 'Terani' (ID: 606266945460764727) by Bingtoolbar
2025-08-04 10:38:18,381 [INFO] Main for user 'Terani' (ID: 606266945460764727) set to 'Demani' by Bingtoolbar
2025-08-04 10:50:52,743 [INFO] logging in using static token
2025-08-04 10:50:53,429 [INFO] Shard ID None has connected to Gateway (Session ID: 141dc85f5b0c3608c7697c1c21145cbe).
2025-08-04 10:53:00,842 [INFO] logging in using static token
2025-08-04 10:53:01,776 [INFO] Shard ID None has connected to Gateway (Session ID: 4b2ba72711c9ad480ba0722c1efdbc0f).
2025-08-04 10:55:34,631 [INFO] logging in using static token
2025-08-04 10:55:35,786 [INFO] Shard ID None has connected to Gateway (Session ID: dbf856d8cfa70c74219f8ff8bb2d3486).
2025-08-04 10:57:10,913 [INFO] logging in using static token
2025-08-04 10:57:11,769 [INFO] Shard ID None has connected to Gateway (Session ID: 219aaf0325e18169f2bf3bcea847dd1e).
2025-08-04 10:57:33,350 [INFO] Main for user 'Terani' (ID: 606266945460764727) set to 'Demani' (Class: None) by Bingtoolbar
2025-08-04 11:00:03,025 [INFO] Main for user 'Terani' (ID: 606266945460764727) set to 'Demani' (Class: Druid) by Bingtoolbar
2025-08-04 11:08:24,899 [INFO] logging in using static token
2025-08-04 11:08:25,798 [INFO] Shard ID None has connected to Gateway (Session ID: 7da1708d2cd4ef49e375d141d7ee4512).
2025-08-04 11:49:24,139 [INFO] Shard ID None has successfully RESUMED session 7da1708d2cd4ef49e375d141d7ee4512.
2025-08-04 13:20:21,598 [INFO] Shard ID None has successfully RESUMED session 7da1708d2cd4ef49e375d141d7ee4512.
2025-08-04 13:38:03,976 [ERROR] Ignoring exception in on_member_join
Traceback (most recent call last):
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/client.py", line 481, in _run_event
    await coro(*args, **kwargs)
  File "/mnt/c/Users/fleet/OneDrive/Documents/coding/discord_bot/guildGateKeeper/bot.py", line 717, in on_member_join
    await send_onboarding_embed(member)
  File "/mnt/c/Users/fleet/OneDrive/Documents/coding/discord_bot/guildGateKeeper/bot.py", line 108, in send_onboarding_embed
    f"For more information, read the rules in {rules_channel.mention}."
AttributeError: 'NoneType' object has no attribute 'mention'
2025-08-04 15:27:06,840 [INFO] Shard ID None has successfully RESUMED session 7da1708d2cd4ef49e375d141d7ee4512.
2025-08-04 16:15:11,674 [INFO] Shard ID None has successfully RESUMED session 7da1708d2cd4ef49e375d141d7ee4512.
2025-08-04 17:16:39,603 [INFO] Shard ID None has successfully RESUMED session 7da1708d2cd4ef49e375d141d7ee4512.
2025-08-04 17:29:17,788 [INFO] logging in using static token
2025-08-04 17:29:18,554 [INFO] Shard ID None has connected to Gateway (Session ID: 91e6aa6f7ba3db9f46a13a8d77c09ba4).
2025-08-04 18:07:53,867 [ERROR] Ignoring exception in command addalt
Traceback (most recent call last):
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/bot.py", line 1366, in invoke
    await ctx.command.invoke(ctx)
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 1021, in invoke
    await self.prepare(ctx)
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 938, in prepare
    await self._parse_arguments(ctx)
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 845, in _parse_arguments
    transformed = await self.transform(ctx, param, attachments)
  File "/home/fleet/.local/lib/python3.10/site-packages/discord/ext/commands/core.py", line 697, in transform
    raise MissingRequiredArgument(param)
discord.ext.commands.errors.MissingRequiredArgument: alt_class is a required argument that is missing.
